### PR TITLE
bt-565: Add ability to view password during signin signup

### LINKED ISSUE #565: Add ability to view password during sign in/up

### PR DESCRIPTION
**Pull request by vladjsp**
closes https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/issues/565

https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/assets/118971905/5932ffbf-7464-4973-8254-6cff2de71a84

### CODE REVIEW

**Review by Simpsonkk — APPROVED** [review]
I can see and hide my password on sign in and sign up pages.

**Review by savinaDaria — APPROVED** [review]

**Comment by nikita-remeslov** [comment]
Conflicts

File: frontend/src/bundles/auth/components/auth-forms/sign-in-form/sign-in-form.tsx [hunk 1]
```diff
@@ -36,6 +46,13 @@ const SignInForm: React.FC<Properties> = ({ onSubmit }) => {
         [handleSubmit, onSubmit],
     );
 
+    const showPasswordIcon = (
+        <PasswordVisibility
+            handleClick={handleClickShowPassword}
+            showPassword={isPasswordVisible}
+        ></PasswordVisibility>
```
Comment (nikita-remeslov) [inline frontend/src/bundles/auth/components/auth-forms/sign-in-form/sign-in-form.tsx]:
```suggestion
        />
```

File: frontend/src/bundles/auth/components/auth-forms/sign-up-form/sign-up-form.tsx [hunk 2]
```diff
@@ -91,6 +98,12 @@ const SignUpForm: React.FC<Properties> = ({ onSubmit }) => {
         </Typography>
     );
 
+    const showPasswordIcon = (
+        <PasswordVisibility
+            handleClick={handleClickShowPassword}
+            showPassword={isPasswordVisible}
+        ></PasswordVisibility>
```
Comment (nikita-remeslov) [inline frontend/src/bundles/auth/components/auth-forms/sign-up-form/sign-up-form.tsx]: the same

**Review by nikita-remeslov — CHANGES_REQUESTED** [review]

File: frontend/src/bundles/auth/components/auth-forms/sign-in-form/sign-in-form.tsx (hunk 1 above)
Comment (vladjsp) [reply frontend/src/bundles/auth/components/auth-forms/sign-in-form/sign-in-form.tsx]: fixed

File: frontend/src/bundles/auth/components/auth-forms/sign-up-form/sign-up-form.tsx (hunk 2 above)
Comment (vladjsp) [reply frontend/src/bundles/auth/components/auth-forms/sign-up-form/sign-up-form.tsx]: fixed

**Review by nikita-remeslov — APPROVED** [review]

**Pull request merged**